app/api/images/images.list.controller.js

```diff
@@ -557,3 +563,131 @@ exports.getStats = utils.route(async (req, res) => {
     rows: result
   });
 });
+
+exports.getImagesBound = utils.route(async (req, res) => {
+  const query = req.query;
+  const attributes = parseAttributes(query);
+  const orderBy = query.sortKey;
+  const orderByNearest = models.sequelize.literal(`images.location <-> st_setsrid(ST_makepoint(${query.longitude}, ${query.latitude}), 4326)`);
+
+  let whereClauses = [];
+
+  if (query.owner_id) {
+    whereClauses.push({ owner_id: inUniqueOrList(query.owner_id) });
+  }
+
+  if (query.collection_id) {
+    whereClauses.push({ collection_id: inUniqueOrList(query.collection_id) });
+  }
+
+  if (query.keyword) {
+    whereClauses.push({
+      [Op.or]: {
+        original_id: iLikeFormatter(query.keyword),
+        title: iLikeFormatter(query.keyword),
+        caption: iLikeFormatter(query.keyword)
+      }
+    });
+  }
+
+  if (query.place_names) {
+    whereClauses.push({ geotags_array: containsUniqueOrList(query.place_names) });
+  }
+
+  if (query.date_shot_min || query.date_shot_max) {
+    whereClauses.push({
+      // Single date
+      [Op.or]: {
+        date_shot: {
+          [Op.and]: {
+            [Op.not]: null,
+            [Op.gte]: query.date_shot_min,
+            [Op.lte]: query.date_shot_max
+          },
+        },
+        // Range of dates
+        [Op.and]: {
+          date_shot_min: {
+            [Op.and]: {
+              [Op.not]: null,
+              [Op.gte]: query.date_shot_min,
+              [Op.lte]: query.date_shot_max
+            }
+          },
+          date_shot_max: {
+            [Op.and]: {
+              [Op.not]: null,
+              [Op.gte]: query.date_shot_min,
+              [Op.lte]: query.date_shot_max
+            }
+          }
+        }
+      }
+    });
+  }
+
+  if (query.view_type) {
+    whereClauses.push({ view_type: inUniqueOrList(query.view_type) });
+  }
+
+  whereClauses.push(
+    Sequelize.where(
+      Sequelize.fn(
+        'ST_Contains',
+        Sequelize.col('images.footprint'),
```

Comment:
The footprint available here is the one compute during the geolocation. There is a more precise one in geometadata table. I think it's better to use it if it is available. Otherwise, use the image one.

Reply:
Updated to check first in geometadata table

```
`CASE 
        WHEN geometadatum.footprint IS NOT NULL AND ST_Contains(geometadatum.footprint, ST_SetSRID(ST_MakePoint(${query.longitude}, ${query.latitude}), 4326)) THEN true
        ELSE ST_Contains(images.footprint, ST_SetSRID(ST_MakePoint(${query.longitude}, ${query.latitude}), 4326))
END`
```